README.md

```diff
@@ -374,6 +374,33 @@ Utilities
 | whoami    |           |        |
 | yes       |           |        |
 
+Targets that compile
+-------
+
+This is an auto-generated table showing which binaries compile for each target-triple. Note that this **does not** indicate that they are fully implemented, or that the tests pass.
+
+|###########################|arch|base32|base64|basename|cat|chgrp|chmod|chown|chroot|cksum|comm|cp|csplit|cut|date|df|dircolors|dirname|du|echo|env|expand|expr|factor|false|fmt|fold|groups|hashsum|head|hostid|hostname|id|install|join|kill|link|ln|logname|ls|mkdir|mkfifo|mknod|mktemp|more|mv|nice|nl|nohup|nproc|numfmt|od|paste|pathchk|pinky|printenv|printf|ptx|pwd|readlink|realpath|relpath|rm|rmdir|seq|shred|shuf|sleep|sort|split|stat|stdbuf|sum|sync|tac|tail|tee|test|timeout|touch|tr|true|truncate|tsort|tty|uname|unexpand|uniq|unlink|uptime|users|wc|who|whoami|yes|
+|---------------------------|----|------|------|--------|---|-----|-----|-----|------|-----|----|--|------|---|----|--|---------|-------|--|----|---|------|----|------|-----|---|----|------|-------|----|------|--------|--|-------|----|----|----|--|-------|--|-----|------|-----|------|----|--|----|--|-----|-----|------|--|-----|-------|-----|--------|------|---|---|--------|--------|-------|--|-----|---|-----|----|-----|----|-----|----|------|---|----|---|----|---|----|-------|-----|--|----|--------|-----|---|-----|--------|----|------|------|-----|--|---|------|---|
+|          aarch64-apple-ios|    |      |      |        |   |     |     |     |      |     |    |  |      |   |    |  |         |       |  |    |   |      |    |      |     |   |    |      |       |    |      |        |  |       |    |    |    |  |       |  |     |      |     |      |    |  |    |  |     |     |      |  |     |       |     |        |      |   |   |        |        |       |  |     |   |     |    |     |    |     |    |      |   |    |   |    |   |    |       |     |  |    |        |     |   |     |        |    |      |      |     |  |   |      |   |
```

Comment:
Are these entirely empty rows due to missing `std` for this platform on your system.

Reply:
Yes, it seems I'm missing the SDK

```
└$ cargo check -p uu_nohup --bin nohup --target=aarch64-apple-ios
error: failed to run `rustc` to learn about target-specific information

Caused by:
  process didn't exit successfully: `rustc - --crate-name ___ --print=file-names --target aarch64-apple-ios --crate-type bin --crate-type rlib --crate-type dylib --crate-type cdylib --crate-type staticlib --crate-type proc-macro --print=sysroot --print=cfg` (exit code: 1)
--- stderr
error: Error loading target specification: failed to get iphoneos SDK path: No such file or directory (os error 2). Use `--print target-list` for a list of built-in targets
```

Reply:
I thought `rustup target add` would suffice for a simple check

Reply:
It work for most (common) targets, but not all.  You can try to parse the error message for stuff like this (although this kinda jank), keep a list of targets that work, or try to set up the missing requirements for the broken targets in the script.  I don’t have a strong preference for which, so it’s up to you.